### PR TITLE
Update index.mdx

### PR DESCRIPTION
**Pull request by drewstone**

### CODE REVIEW

**Comment by drewstone** [comment]
@dutterbutter any idea why this fails CI?

**Comment by dutterbutter** [comment]
@drewstone I will investigate right now

File: v1/docs/09-applications/c-asset-protocol/index.mdx [hunk 1]
```diff
@@ -7,134 +7,94 @@ category: ispp
...
+- **Blinding randomness**
+    - A random value that provides additional randomization to the hash commitment.
 
-Notice that the invariant `sum of inputUTXOs + publicAmount = sum of outputUTXOs` holds in each of the steps. Since this invariant covers depositing and withdrawing, we do not need separate functions for each but rather a single `transact` function which takes care of both depositing and withdrawing. When Alice calls the `transact` function on the Tornado Pool contract she will submit a proof that (amongst other things) the invariant holds for her proposed transaction.
+The ***shielded*** component comes from the hash commitment to the UTXO data. We take the commitment to be the Poseidon hash function of the underlying data.
 
-## Exact UTXO Structure
+$$
+\textbf{commitment} = Poseidon(\text{chain}_\text{id},\text{amount}, \text{public}_\text{key}, \text{blinding})
+$$
```
Comment (dutterbutter) [inline v1/docs/09-applications/c-asset-protocol/index.mdx:55]: It fails to understand this formatting @drewstone

**Comment by dutterbutter** [comment]
@drewstone I have updated the formatting, it should build correctly now.

**Pull request merged**